### PR TITLE
Feature/add forced selected id dropdown

### PR DESCRIPTION
**Pull request by mlunoe**

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @daiyiwhale

**Comment by jfurrow** [comment]
:ship: :ship: :ship: :ship: :ship: :ship: :ship: :ship: :ship: :ship: :ship: :ship: :ship: :ship: :ship:

File: src/Dropdown/Dropdown.js [hunk 1]
```diff
@@ -28,6 +28,13 @@ export default class Dropdown extends Util.mixin(BindMixin) {
     };
   }
 
+  componentWillReceiveProps(nextProps) {
+    // Set the selectedID to the forceSelectedID if set
+    if (this.props.forceSelectedID) {
+      this.setState({selectedID: nextProps.forceSelectedID});
+    }
```
Comment (rcorral) [inline src/Dropdown/Dropdown.js]: Why store it in state? If it's already in props can't we just always look there?

File: src/Dropdown/Dropdown.js [hunk 2]
```diff
@@ -57,9 +64,16 @@ export default class Dropdown extends Util.mixin(BindMixin) {
   }
 
   componentWillMount() {
-    this.setState({
-      selectedID: this.props.selectedID
-    });
+    let props = this.props;
+    if (props.forceSelectedID) {
+      this.setState({
+        selectedID: props.forceSelectedID
```
Comment (rcorral) [inline src/Dropdown/Dropdown.js]: Same here, I think we just need the `else` part of this if.

File: src/Dropdown/Dropdown.js (hunk 1 above)
Comment (jfurrow) [inline src/Dropdown/Dropdown.js]: :+1: Good point

Comment (mlunoe) [inline src/Dropdown/Dropdown.js]: PR is up here: https://github.com/mesosphere/reactjs-components/pull/123

**Pull request merged**